charts/citylens-routes-ui/templates/deployment.yaml

```
@@ -0,0 +1,89 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "citylens-ui.name" . }}
+  {{- with .Values.service.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  {{- with .Values.service.labels }}
+  labels:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  replicas: {{ .Values.replicas }}
+  revisionHistoryLimit: {{ .Values.revisionHistoryLimit }}
+  {{- if .Values.strategy }}
+  strategy:
+    {{- toYaml .Values.strategy | nindent 4 }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "citylens-ui.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      annotations:
+        {{- with .Values.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+      labels:
+        {{- include "citylens-ui.labels" . | nindent 8 }}
+        {{- with .Values.podLabels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      enableServiceLinks: false
+      containers:
+        - name: {{ include "citylens-ui.name" . }}
+          image: "{{ required "A valid .Values.dgctlDockerRegistry entry required" .Values.dgctlDockerRegistry }}/{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.imagePullPolicy }}
+          ports:
+            - name: http
+              containerPort: {{ .Values.containerPort }}
+          lifecycle:
+            preStop:
+              exec:
+                command: [ "/bin/sh", "-c", "sleep 5" ]
+          {{- if .Values.resources }}
+          resources: {{- toYaml .Values.resources | nindent 12 }}
+          {{- end }}
+          env:
+            - name: CATALOG_API_URL
+              value: {{ required "A valid .Values.env.CATALOG_API_URL entry required" .Values.env.CATALOG_API_URL | quote }}
+            - name: MAPGL_API_URL
+              value: {{ required "A valid .Values.env.MAPGL_API_URL entry required" .Values.env.MAPGL_API_URL | quote }}
```

давай тут добавим подстановку пути, а в переменной `MAPGL_API_URL`  будем передавать просто урл
```suggestion
              value: {{ printf "%s/api/js" (required "A valid .Values.env.MAPGL_API_URL entry required" .Values.env.MAPGL_API_URL) | quote }}
```

---

чтобы минимизировать риск ошибок, если в других чартах указан просто url для mapgl, то и тут человек напишет просто url, ну и мы хардкодим на уровне чарта, а не приложения, если путь поменяется - без проблем поменяем тут

---

Поправил